deps/build.jl

```diff
@@ -0,0 +1,3 @@
+using Blink
+
+Blink.AtomShell.install()
```

Comment:
This whole file can be deleted because Electron.jl handles the necessary binary dependency installation.